sceneformsrc/sceneform/src/main/java/com/google/flatbuffers/FlatBufferBuilder.java

```diff
@@ -16,8 +16,6 @@
 
 package com.google.flatbuffers;
```

Comment:
Isn't the presence of this class going to still clash with other Sceneform versions?